requirements.txt

```diff
@@ -1,3 +1,4 @@
+argparse
 termcolor
 tornado
-pre-commit
\ No newline at end of file
+pre-commit
```

Comment:
looks like development requirement. As I see, we do not divide dev and prod requirements. Should we?

Reply:
@FallenChromium in case, when we removed `pre-commit` from requirements, what will we do if we need to add another dev dependency